NEWS

```diff
@@ -1,3 +1,11 @@
+pghoard X.X.X (2016-XX-XX)
+==========================
+
+* Support for `local-tar` backups of a replica on PostgreSQL 9.6 and newer
+  using the new non-exclusive backup mechanism.  Previous PostgreSQL
+  versions require the pgespresso extension to take backups of replicas
```

Comment:
to take "local-tar" backups of replicas. (since we've been able to take pg_basebackup backups forever.